spec/models/like_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  it 'Update like counter for post' do
```

Comment:
- [ ] Well-written tests so far for your models' validations. However, you've missed a small technical point to implement, and it is the `describe` method for describing each test case as a unique entity in order to make it easier for the developers to read and track the code.
see an example!
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```
_Please implement the changes in all `_spec` files_

Reply:
Thank you so much, I have made the changes.